### PR TITLE
delete blank project and fix issues #161, #164, #165

### LINKED ISSUE #161: Missing public key for -ID- in default
On log-in I get the following return message
“Please enter the accountId that you logged in with: -myID-
Missing public key for -myID- in default
Logged in with -myID-”

What is the expected behaviour? Is the user supposed to add the public key; if yes, how? Is this message needed?

### PR DESCRIPTION
**Pull request by icerove**
fixes #161, #164, #165

now when logged in successfully, it will show Logged in as {accountId} with public key {public-key (ed25519:CV16URecQ1jshXQxzBRZStywfFBCuAVFyFYk39ngohuh)} successfully
user can copy correct public keys and not copy from url.

also reformat the code in bin/near.js and index.js into more clear format.
rewrite readme to lastest version of near-shell

### CODE REVIEW

**Comment by icerove** [comment]
test is not passing. WIP for tests

**Comment by icerove** [comment]
now readme is like this:
### Commands

For account:
```bash
  near login                                   # create a developer account
  near create_account <accountId>              # create a developer account with masterAccount, publicKey and initialBalance
  near view <accountId>                        # view account state
  near keys <accountId>                        # view account public keys
  near send <sender> <receiver> <amount>       # send tokens to given receiver
  near stake <accountId> <publicKey> <amount>  # create staking transaction (base58 encoded)
  near delete <accountId> <beneficiaryId>      # delete an account and transfer funds to beneficiary account
```

For smart contract:
```bash
  near build                                   # build your smart contract
  near deploy                                  # deploy your smart contract
  near call <contractName> <methodName>        # schedule smart contract call which
  [args]                                       # can modify state
  near view <contractName> <methodName>        # make smart contract call which can
  [args]                                       # view state
  near clean                                   # clean the build environment
```

For transactions:
```bash
  near tx-status <hash>                        # lookup transaction status by hash
```

### Options

| Option                    | Description                                   | Type      | Default               |
| --------------------------|:---------------------------------------------:| :---------|:----------------------|
| --help                    | Show help                                     | [boolean] |                       |
| --version                 | Show version number                           | [boolean] |                       |
| --nodeUrl, --node_url     | NEAR node URL                                 | [string]  |"http://localhost:3030"|
| --networkId, --network_id | NEAR network ID for different keys by network | [string]  |"default"              |
| --helperUrl               | NEAR contract helper URL                      | [string]  |                       |
| --keyPath                 | Path to master account key                    | [string]  |                       |
| --homeDir                 | Where to look for master account              | [string]  |"~/.near"              |
| --accountId, --account_id | Unique identifier for the account             | [string]  [required]|                       |
| --masterAccount           | Account used to create requested account.     | [string]  [required]|                       |
| --publicKey               | Public key to initialize the account with     | [string]  [required]|                       |
| --initialBalance          | Number of tokens to transfer to newly account | [string]  [required]|                       |

**Comment by bowenwang1996** [comment]
In readme, for staking, `create staking transaction (base58 encoded)` is a bit confusing. It is not clear whether it is saying that the transaction is base58 encoded or the key is base58 encoded. Also what does `near clean` do exactly? Maybe the comment could be more clear on what is cleaned.

File: bin/near [hunk 1]
```diff
@@ -118,8 +125,8 @@ const viewAccount = {
 };
 
 const deleteAccount = {
-  command: 'delete_account <accountId> <beneficiaryId>',
-  desc: 'delete an account and transfer funds to beneficiary account.',
+  command: 'delete <accountId> <beneficiaryId>',
```
Comment (bowenwang1996) [inline bin/near]: why is the command changed?

File: bin/near [hunk 2]
```diff
@@ -146,52 +153,16 @@ const keys = {
...
-      desc: 'project directory',
-      type: 'string',
-      default: '.'
-    }),
-  handler: (argv) => exitOnError(main.newProject(argv))
-};
-
 const stake = {
-  command: 'stake [accountId] [publicKey] [amount]',
+  command: 'stake <accountId> <stakeKey> <amount>',
```
Comment (bowenwang1996) [inline bin/near]: `stakingKey` makes more sense to me

File: bin/near [hunk 3]
```diff
@@ -12,6 +12,25 @@ const exitOnError = async function(promise) {
   }
 }
 
+//For smart contract:
```
Comment (vgrichina) [inline bin/near]: nit: let's have spaces after `//` 
probably should configure eslint to fix this

File: bin/near (hunk 1 above)
Comment (vgrichina) [reply bin/near]: let's have shorter command, I like it

File: bin/near (hunk 2 above)
Comment (icerove) [reply bin/near]: is staking a word that means the process not the result? the command is stake should it really be staking make more sense?

**Pull request merged**